build/CreateCBSVPack.ps1

```diff
@@ -13,6 +13,8 @@ Trap
     Exit 1
 };
 
+$repoRoot = $script:MyInvocation.MyCommand.Path | Split-Path -Parent | Split-Path -Parent
```

Comment:
Seems a bit fragile on the project directory structure. I'm not sure if there is an alternative though.

PowerShell 3.0 has MyInvocation.PSScriptRoot which might be better?

Reply:
Yeah, it's not a great solution, but I don't know of a better approach. This pattern is used across the repo for many of the scripts.

I suppose one solution would be to search up the chain of parent directories, looks for a directory containing a ".git" directory. But I am going to leave this as it is for now.